pkg/connectors/datacatalog/clients/datacatalog_grpc.go

```diff
@@ -220,3 +220,9 @@ func (m *grpcDataCatalog) CreateAsset(in *datacatalog.CreateAssetRequest, creds
 func (m *grpcDataCatalog) DeleteAsset(in *datacatalog.DeleteAssetRequest, creds string) (*datacatalog.DeleteAssetResponse, error) {
 	return &datacatalog.DeleteAssetResponse{Status: "DeleteAsset not implemented via GRPC"}, nil
 }
+
```

Comment:
When will grpc support be removed? Is there an issue for that?

Reply:
Recently we used Egeria connector (with grpc support) for a demo. I am not sure if we should remove grpc support without adding REST API support to Egeria Connector. Please advise. Thanks.